virtualizarr/translators/kerchunk.py

```diff
@@ -222,9 +220,17 @@ def find_var_names(ds_reference_dict: KerchunkStoreRefs) -> list[str]:
     """Find the names of zarr variables in this store/group."""
 
     refs = ds_reference_dict["refs"]
-    found_var_names = {key.split("/")[0] for key in refs.keys() if "/" in key}
 
-    return list(found_var_names)
+    found_var_names = []
+    for key in refs.keys():
+        # has to capture "foo/.zarray", but ignore ".zgroup", ".zattrs", and "subgroup/bar/.zarray"
+        # TODO this might be a sign that we should introduce a KerchunkGroupRefs type and cut down the references before getting to this point...
+        if key not in (".zgroup", ".zattrs", ".zmetadata"):
```

Comment:
@norlandrhagen the only reason I needed the `".zmetadata"` key in the check here is to make one kerchunk parquet test pass. Is it really the case that kerchunk parquet references use different keys than kerchunk json references? Or is that just a mistake in the fake kerchunk parquet data that we create and use in 

https://github.com/zarr-developers/VirtualiZarr/blob/af9c3747896b6faff1a3a670b5c485e509659405/virtualizarr/tests/test_writers/test_kerchunk.py#L137

Reply:
Hmm, I think we were relying on the parquet directory to have a `.zmetadata` file so that we could identify the directory containing parquet's as parquet. https://github.com/zarr-developers/VirtualiZarr/pull/278#discussion_r1821097143

Reply:
Okay let's merge then.

Reply:
Oh I have another bug to fix first...